seumich/views.py

```diff
@@ -211,6 +211,11 @@ def get_queryset(self):
 class IndexView(LoginRequiredMixin, UserLogPageViewMixin, View):
 
     def get(self, request):
+        # If the user has no mentors (is an admin) just redirect to main list
+        try:
+            has_mentor = Mentor.objects.get(username=request.user.username)
+        except Mentor.DoesNotExist:
+            return redirect('seumich:advisors_list')
```

Comment:
Can we put a log message before redirect?

Reply:
Sure! This was catching and returning a 404 later for both prod and DEV, now it redirects to a list that should always work.

Reply:
Log statement added